### DISCUSSION TITLE
Backup tags for people from Google Takeout

### DISCUSSION THREAD

**Discussion by vishnukvmd** [ente-io/ente#605] [Enhancements]
Google Takeout provides tags for people within a photo.

These could be attached to the file's metadata, as a special type of tag (`people`), and once we implement facial recognition, we could let users associate these tags to clusters.

**Comment by abhinavkgrd** [comment]
Should we start importing them and storing them in metadata from now itself? @vishnukvmd

**Comment by vishnukvmd** [comment]
I think we should wait until we've finalized the data structure for face embeddings and their clusters, at which point the linking will be more meaningful. But if you or @ua741 see value in shipping it before that, go for it. As long as we ensure that this has a type that's separate from manual-tags, we should be good.

**Comment by garyp** [comment]
Any plans to implement this, now that Ente supports tagging people?

**Comment by Hemanth-Eduru** [comment]
@vishnukvmd @ua741 Adding to this is it possible that Ente Export contains the people info tags (if not already) so that it can be even used to load the data again without re indexing .. This could be useful in case of crash recovery, where I export photos to a NAS.
Where as ente server was on local storage
